### PR TITLE
removed normalize-before

### PR DESCRIPTION
**Pull request by olafbrah**
-removed references to the normalize-before parameter
-uncommented default to cpu because I am using a mac

### CODE REVIEW

File: egs/ema/voc1/local/predict_wav.py [hunk 1]
```diff
@@ -114,15 +105,15 @@ def main():
...
+    if torch.cuda.is_available():
+        device = torch.device("cuda")
+    else:
+        device = torch.device("cpu")
     model = load_model(args.checkpoint, config)
     logging.info(f"Loaded model parameters from {args.checkpoint}.")
-    if args.normalize_before:
-        assert hasattr(model, "mean"), "Feature stats are not registered."
-        assert hasattr(model, "scale"), "Feature stats are not registered."
+    # if args.normalize_before:
```
Comment (peter-yh-wu) [inline egs/ema/voc1/local/predict_wav.py]: These can be deleted too

File: egs/ema/voc1/local/predict_wav.py [hunk 2]
```diff
@@ -135,17 +126,17 @@ def count_parameters(model):
     input_lens = []
     with torch.no_grad():
         for fid, featp in tqdm(zip(fids, featps), total=len(fids)):
-            c = np.load(featp)
+            c = np.load(featp, allow_pickle=True)
```
Comment (peter-yh-wu) [inline egs/ema/voc1/local/predict_wav.py]: allow_pickle not needed

**Review by peter-yh-wu — CHANGES_REQUESTED** [review]

**Review by olafbrah — COMMENTED** [review]
added fixes

**Review by peter-yh-wu — APPROVED** [review]
